neuro_san/interfaces/agent_session.py

```diff
@@ -14,20 +14,14 @@
 from typing import Dict
 from typing import Generator
 
+from neuro_san.interfaces.agent_session_constants import AgentSessionConstants
 
-class AgentSession:
+
+class AgentSession(AgentSessionConstants):
     """
     Interface for the initiation and continuity of a single Agent session.
     """
 
-    # Default port for the Agent Service
-    # This port number will also be mentioned in its Dockerfile
-    DEFAULT_PORT: int = 30011
-
-    # Default port for the Agent HTTP Service
-    # This port number will also be mentioned in its Dockerfile
-    DEFAULT_HTTP_PORT: int = 8080
```

Comment:
These move to new AgentSessionConstants interface so they can be defined in a single place.